### PR TITLE
Quick docs fixes

### PR DESCRIPTION
**Pull request by Aaryan-R-S**
# Description

1. Add a note to clear up the local storage for authSig so that you can avoid errors from Lit nodes because of stale authSig
2. All of the advanced topics pages also have the same copy ('Learn about the advanced topics for Compute (Lit Actions) in Lit SDK!')
3. The second code snippet for [Obtain an auth sig](https://developer.litprotocol.com/v3/sdk/access-control/quick-start#obtain-an-auth-sig) is incomplete (missing main function and has a dangling } at the end)
Same for the code snippet in [this section too](https://developer.litprotocol.com/v3/sdk/access-control/quick-start#obtain-an-auth-sig-1)
4. Add the docs feedback link to the bottom of all articles
5. Update footer links
6. Remove "replace" step for signature and addresses of "0x" by "" ([here1](https://developer.litprotocol.com/v3/concepts/capacity-credits-concept) and [here2](https://developer.litprotocol.com/v3/sdk/capacity-credits)) 
7. Make Minting Capacity Credits and Delegate Usage a sub section of Performing Decryption as in [this decryption section](https://developer.litprotocol.com/v3/sdk/access-control/quick-start#performing-decryption), it seems like the code snippet to perform the decryption is missing.
8. Add code snippet for setting up Lit Auth Client in "Install the requires packages" and "Mint a PKP and Add Permitted Scopes" section
9. Fix spacing in code snippets

# Fixes
- [LIT-2918](https://linear.app/litprotocol/issue/LIT-2918/add-a-note-for-clearing-up-local-storage-for-authsig)
- [LIT-2963](https://linear.app/litprotocol/issue/LIT-2963/correct-quick-start-code-snippets)
- [LIT-2950](https://linear.app/litprotocol/issue/LIT-2950/add-the-docs-feedback-link-to-the-bottom-of-all-articles)
- [LIT-3052](https://linear.app/litprotocol/issue/LIT-3052/remove-this-replace-step-for-signature-and-addresses)
- [LIT-3050](https://linear.app/litprotocol/issue/LIT-3050/make-minting-capacity-credits-and-delegate-usage-a-sub-section-of)
- [LIT-3051](https://linear.app/litprotocol/issue/LIT-3051/make-quick-start-guides-more-uniform-and-fix-some-issues)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Introducing new feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

Link to the relevant updated sections in the Netlify preview

- [ ] Link 1: [description of change]
- [ ] Link 2: [...]

# Checklist:

General
- [ ] I have performed a self-review of my code
- [ ] I have fixed all grammar issues (can use an AI tool to check), and explanations are in active voice
- [ ] I have checked the additions are concise
- [ ] Language is consistent with existing documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published (ie. SDK changes, node dependencies)

If I have added a new concept, I have
- [ ] included a beginner friendly explanation
- [ ] included a basic technical introduction and code sample
- [ ] new terms are defined, both in relevant new pages and in the glossary

### CODE REVIEW

**Review by glitch003 — CHANGES_REQUESTED** [review]
hey, apologies for the change request, but instead of pasting in the text below in a bunch of places, can you put it in one file and then import it where it needs to be displayed?  https://docusaurus.io/docs/markdown-features/react#importing-markdown

```
Not finding the answer you're looking for? Share your feedback on these docs by creating an issue in our [GitHub Issues and Reports](https://github.com/LIT-Protocol/Issues-and-Reports/issues) repository or get support by visiting our [Support](/v3/support/intro) page. We are here to help you!
```

This will make it a lot easier if we need to change it someday because we can change it in one place instead of in a lot of places

**Comment by glitch003** [comment]
the rest of this PR looks good tho!

**Comment by Aaryan-R-S** [comment]
Hey @glitch003!
Actually almost all of the articles are .md files not .mdx files. I looked up internet but I don't think we can do that.
Do you have any other suggestions?
Either we can:
1. Go and accept this one (we can still change this easliy by writing a program, infact I did this using a python program only)
2. Convert all .md files to .mdx and then include
3. Skip adding this feedback thing as ultimately we have support thing in global footer

**Comment by glitch003** [comment]
The import syntax should work with .md files.  The docusaurus docs say they treat .md and .mdx files the same: https://docusaurus.io/docs/markdown-features#mdx-vs-commonmark

**Comment by Aaryan-R-S** [comment]
Thanks @glitch003 for the suggestion. I did it :) 
PR is up for review now!

File: src/components/FeedbackComponent.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const FeedbackComponent = () => {
+    return (
+        <div>
+            Not finding the answer you're looking for? Share your feedback on these docs by creating an issue in our <a href="https://github.com/LIT-Protocol/Issues-and-Reports/issues">GitHub Issues and Reports</a> repository or get support by visiting our <a href="/v3/support/intro">Support</a> page. We are here to help you!
```
Comment (a1ttech) [inline src/components/FeedbackComponent.jsx:6]: can we make this a callout box (like the one at the top of this page: https://developer.litprotocol.com/v3/what-is-lit) just so it stands out a bit more?

i think we can also remove the end 'we are here to help you' text

Comment (Aaryan-R-S) [reply src/components/FeedbackComponent.jsx]: Nice suggestion @a1ttech!
Fixed :)

**Review by a1ttech — APPROVED** [review]

**Pull request merged**